model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -84,7 +84,14 @@ data class ResolvedLicense(
      * All text locations where this license was found.
      */
     val locations: Set<ResolvedLicenseLocation>
-)
+) {
+    /**
+     * True, if this license was [detected][LicenseSource.DETECTED] and all [locations] have matching path excludes.
+     */
+    val isExcluded by lazy {
+        LicenseSource.DETECTED in sources && locations.all { it.matchingPathExcludes.isNotEmpty() }
```

Comment:
I find it a bit tricky to get the `DETECTED in sources`bit, and how it is supposed to be used.
`isExcluded` seems to make sense only for detected licenses, do you see a way to clarify this a bit more?
E.g. what do you think about: `isDetectedExcluded` or `hasNonExcludedFileFindings`, `hasNonExcludedDetectedLicense`

Reply:
Doesn't the documentation of the function make that pretty clear?

Reply:
Imo no. What does the documentation tell about the case `souces = { DETECTED, DECLARED }?

Reply:
I have renamed it to `isDetectedExcluded`.

Reply:
Thanks!